Gemfile.lock

```diff
@@ -250,25 +245,25 @@ GEM
     method_source (0.9.2)
     mime-types (3.3)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.0904)
+    mime-types-data (3.2019.1009)
     mimemagic (0.3.3)
     mini_magick (4.9.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.12.2)
+    minitest (5.13.0)
     msgpack (1.3.1)
-    mysql2 (0.5.2)
+    mysql2 (0.5.3)
     net-scp (2.0.0)
       net-ssh (>= 2.6.5, < 6.0.0)
     net-ssh (5.2.0)
     netrc (0.11.0)
     nio4r (2.5.2)
-    nokogiri (1.10.4)
+    nokogiri (1.10.7)
       mini_portile2 (~> 2.4.0)
-    oj (3.9.1)
+    oj (3.10.0)
     orm_adapter (0.5.0)
-    parallel (1.17.0)
-    parser (2.6.4.1)
+    parallel (1.19.1)
+    parser (2.6.5.0)
```

Comment:
### [parser](https://github.com/whitequark/parser)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.6.4.1 → 2.6.5.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/whitequark/parser/compare/v2.6.4.1...v2.6.5.0).

These are the individual commits:
- (3264d5d) [Update changelog.](https://github.com/whitequark/parser/commit/3264d5d3a1738e405d43d2d57101881d9d40a741)
- (c1295b0) [Detect and warn about empty else (#611)](https://github.com/whitequark/parser/commit/c1295b054cea829282bc1f8b3c0a7581c85c6e70)
- (d2444ae) [- lexer.rl: emit tMETHREF as tDOT+tCOLON for rubies < 27. (#…](https://github.com/whitequark/parser/commit/d2444aef18cc1da719fbe2592f22612702b76815)
- (f8b99f5) [+ lexer.rl: support comments before leading dot in 27 mode. …](https://github.com/whitequark/parser/commit/f8b99f5c59564a6e3eae5b3e9a6db2e6b9e1c013)
- (1523bcc) [Suppress keyword argument warnings in Ruby 2.7 (#612)](https://github.com/whitequark/parser/commit/1523bcc94ca8939d2426d2717c94305d4cb80505)
- (24654a8) [+ lexer.rl: changed max numparam to `@9` (#617)](https://github.com/whitequark/parser/commit/24654a8987729e8269a36a1cbce9a5be7899e7cb)
- (c98f161) [* Bump ruby versions to 2.4.9, 2.5.7 and 2.6.5. (#619)](https://github.com/whitequark/parser/commit/c98f1618a5b965a32dab7e1b37043202ba83dd4d)
- (6b33751) [Bump version.](https://github.com/whitequark/parser/commit/6b33751267164a217a7a2ad66ef132592818e716)


</details>